go/lang/correctness/overflow/overflow.go

```diff
@@ -16,7 +16,7 @@ func mainInt16Ex1() {
 }
 
 func mainInt16Ex2() {
-	// ok
+	// ok: interter-overflow-int16
```

Comment:
Huh, that should probably be spelled `integer` in the rule ID, right? Mind fixing that typo in the rule?

Reply:
Wilco, thanks!
```suggestion
	// ok: integer-overflow-int16
```

Reply:
Note it needs to be updated in the YAML as well to match! Probably worth creating a new PR for that.